src/components/sortable.jsx

```diff
@@ -255,7 +255,7 @@ var Sortable = React.createClass({
         layout: React.PropTypes.oneOf([HORIZONTAL, VERTICAL]),
         padding: React.PropTypes.bool,
         disabled: React.PropTypes.bool,
-        constraints: React.PropTypes.object,
+        constraints: React.PropTypes.shape({}),
```

Comment:
I think I would prefer these were replaced with `React.PropTypes.any` if we don't know what the contents are. This seems to imply that they should take an object with no properties, or that we don't use any of the properties, whereas `any` says we don't know.